File: 1193.py
num = int(input())
n = 1
ans = 0
while(1):
    if num == 1:
        print(str(1) + '/' + str(1))
        break
    if n*(n+1)/2 >= num:
        ans = num - (n-1)*n/2
        if n%2==0:
            print(str(int(ans)) + '/' + str(int(n-ans+1)))
            break
        else:
            print(str(int(n-ans+1)) + '/' + str(int(ans)))
            break
    n += 1

File: 1193.c
#include <stdio.h>

int main(){
	int num;
	int i;
	int left, right;
	
	scanf("%d",&num);
	i=2;
	if(num==1){
		printf("1/1");
		return 0;
	}
	while(1){
		if((num<=((i*i)+i)/2) && (num > ((i*i)-i)/2)){
			if(i%2 == 0){
				left = num-((i*i)/2 - i/2);
				right = i+1 - left;
				printf("%d/%d",left,right);
				break;
			}else{
				right = num-((i*i)/2 - i/2);
				left = i+1 - right;
				printf("%d/%d",left,right);
				break;
			}
		}
		i++;
	}
	
	return 0;
}

